### PR TITLE
CID process perfect

### PR DESCRIPTION
**Pull request by 286545761**

### CODE REVIEW

**Review by Zero0000O — APPROVED** [review]

File: ELA/Wallet/walletManage/DID/view/HWMallDocumentsListTableViewCell.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//
```
Comment (heropan) [inline ELA/Wallet/walletManage/DID/view/HWMallDocumentsListTableViewCell.h:1]: license

File: ELA/Wallet/walletManage/DID/view/HWMallDocumentsListTableViewCell.m [hunk 2]
```diff
@@ -0,0 +1,23 @@
+//
```
Comment (heropan) [inline ELA/Wallet/walletManage/DID/view/HWMallDocumentsListTableViewCell.m:1]: license

File: ELA/found/SuperPoint/view/HWMCrCommitteeInformationHeaderView.h [hunk 3]
```diff
@@ -0,0 +1,21 @@
+//
```
Comment (heropan) [inline ELA/found/SuperPoint/view/HWMCrCommitteeInformationHeaderView.h:1]: license

File: ELA/found/SuperPoint/view/HWMCrCommitteeInformationHeaderView.m [hunk 4]
```diff
@@ -0,0 +1,57 @@
+//
+//  HWMCrCommitteeInformationHeaderView.m
+//  elastos wallet
+//
+//  Created by  on 2020/3/26.
+//
```
Comment (heropan) [inline ELA/found/SuperPoint/view/HWMCrCommitteeInformationHeaderView.m:6]: license

File: ELA/Base/English.lproj/Localizable.strings [hunk 5]
```diff
@@ -0,0 +1,104 @@
+/* 
+  Localizable.strings
```
Comment (heropan) [inline ELA/Base/English.lproj/Localizable.strings:2]: license

**Pull request merged**